linetools/scripts/lt_xspec.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+
+"""
+Plot a spectrum with an interactive QT GUI
+"""
+import pdb
+import sys
+
+from PyQt4 import QtGui
+
+from linetools.guis.xspecgui import XSpecGui
+
```

Comment:
Please put the PyQt4 and xspecgui imports inside main(), to avoid imports before parsing the arguments.